DOCSP-48161: Async examples for Connect pages (#236)
mongodb/docs-pymongo: source/connect/mongoclient.txt

   uri = "mongodb://localhost:27017/"
   client = MongoClient(uri)

> If you must use an asynchronous version of the ``MongoClient`` class, use the

Reviewer: What is this sentence aiming to communicate? It's a little unclear to me what "if you must use an asynchronous version...` means.

Author: Can tweak this!

> If you need to use an asynchronous version of the ``MongoClient`` class, use the

Reviewer 2: ```suggestion
If you must use an asynchronous version of the ``MongoClient`` class, use the
```
